test/unittest/unittest_nntrainer_lr_scheduler.cpp

```diff
@@ -478,6 +479,72 @@ TEST(lr_cosine, getlearningrate_01_n) {
   EXPECT_NO_THROW(lr->finalize());
   EXPECT_FLOAT_EQ(lr->getLearningRate(0), 1);
 }
+
+TEST(lr_linear, prop_01_n) {
+  auto lr = createLRS("linear");
+  EXPECT_THROW(lr->setProperty({"unknown=unknown"}), std::invalid_argument);
+}
+
+TEST(lr_linear, prop_02_n) {
+  auto lr = createLRS("linear");
+  EXPECT_THROW(lr->setProperty({"learning_rate:0.1"}), std::invalid_argument);
+}
+
+TEST(lr_linear, prop_03_n) {
+  auto lr = createLRS("linear");
+  EXPECT_NO_THROW(lr->setProperty({"max_learning_rate=1.0"}));
+}
```

Comment:
`_n` stands for negative here. please check test cases and change it to `_p` if it's a positive test case!

Reply:
Fixed it. Thank you!